openpype/hosts/blender/plugins/publish/extract_blend_animation.py

```diff
@@ -21,7 +21,12 @@ def process(self, instance):
         self.log.info("Performing extraction...")
 
         # Deactivate preview range
-        bpy.context.scene.use_preview_range = False
+        bpy.types.Scene.has_preview_range = bpy.props.BoolProperty(
```

Comment:
You don't need such complicated design...
Simply keep the value in a boolean and restore it at the end of the same extraction...

Reply:
I tried, but it turns preview range back on too soon and results in shortened playblasts. Eventually I could try moving the implementation to `integrate kitsu review` to keep it in a single file, if you think it's better

Reply:
Ah okay, therefore yes, the simpler the better!